src/Hl7.Fhir.Core.Tests/Model/ModelTests.cs

```diff
@@ -760,5 +760,12 @@ public void TestHumanNameFluentInitializers()
             pat.Name.FirstOrDefault().Family.Should().Be("Levin");
             pat.Name.FirstOrDefault().Suffix.Should().ContainSingle("The 7th");
         }
+
+        [TestMethod]
+        public void TestCorrectCurrencyDescription()
```

Comment:
Can you add some instructions in any case this unittest fails? Because after generating the pocos we have to manually correct the Money class.